gatsby-config.js

```diff
@@ -87,5 +87,15 @@ module.exports = {
     {
       resolve: 'gatsby-plugin-extract-schema',
     },
+    // This adds the Google Analytics pixel
+    {
+      resolve: 'gatsby-plugin-google-tagmanager',
+      options: {
+        id: 'GTM-KMVFH7G',
```

Comment:
just for my understanding, where did this ID come from?

Reply:
I had asked @kelseyevans 😬 , so I'm assuming something she can see in Google Analytics, but I'll follow-up